MovieApp/MovieApp/Presenters/HomePagePresenter.swift

```diff
@@ -1,42 +1,16 @@
 import Foundation
 
 final class HomePagePresenter {
-    
-    var movies: [Movie] {
-        [
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2007)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf"),
-            Movie(
-                title: "Iron Man (2008)",
-                description: "After being held captive in an Afghan cave, billionaire engineer Tony Stark creates a unique weaponized suit of armor to fight evil.",
-                imagePath: "MockupMovieImage.pdf")
-        ]
+        
+    func presentPopularMovies(completionHandler: @escaping (Result<[Movie]?, RequestError>) -> Void) {
```

Comment:
This function doesn't present popular movies, it fetches them so a better name would be `getPopularMovies` and completion handler can be named just `completion`.